WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -354,6 +354,11 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 	protected function is_local_data_stream( $clean_param_value ) {
 
 		$stripped = TextStrings::stripQuotes( $clean_param_value );
+
+		if ( strpos( $clean_param_value, '\\' ) === 0 ) {
+			$clean_param_value = substr( $clean_param_value, 1 );
+		}
```

Comment:
Could this be simplified ? `ltrim( $clean_param_value, '\\')` ?

Reply:
Yes, I don't know why this did not occur to me when preparing this fix. It is done now. Thanks for the suggestion.